Assets/Scripts/Isamu/Utils/YieldRegistry.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+public static class YieldRegistry
```

Comment:
This is another static helper class to avoid garbage collection for `YieldInstruction` objects (i.e. `new WaitForSeconds(1f)`). It caches and reuses `YieldInstruction` objects so we don't have to create new ones every time. Currently unused but I'm adding so we have it handy.